test/fixtures/oooRequest/oooRequest.ts

```
@@ -27,21 +27,21 @@ export const invalidOooStatusRequests = {
   reason: "Approval granted.",
 };
 
 export const updateOooRejectedRequests = {
-  state: REQUEST_STATE.REJECTED,
+  state: REQUEST_STATUS.REJECTED,
   lastModifiedBy: "admin123",
   updatedAt: 1234567890,
   reason: "Sorry, we can't approve additional leave at this time.",
 };
 
 export const validOooStatusUpdate ={
```

ESLint

[error] Insert `·`

(prettier/prettier)